src/index-nomodules.css

```diff
@@ -226,3 +226,143 @@ a.disabled {
 textarea {
   resize: vertical;
 }
+
+:root {
```

Comment:
Looking at the patternfly-react-seed app base App component (https://github.com/patternfly/patternfly-react-seed/blob/master/src/app/index.tsx), the PF4-react `base.css` import is done there.

We do something similar in index.js:
https://github.com/oVirt/ovirt-web-ui/blob/b7351eb672bfab8fe3d3c10ff832a97e9a958c62/src/index.js#L16-L19

Can this edit be avoided by adding the PF4-react import below the PF3-react import?  If there are problems with that, maybe there is a way to use patternfly4's `patternfly-no-reset.css` as the base (via the PF4-react code preferably)?

I would prefer to keep long bits like this out of our code as much as possible.  Feels like it would make branding all that more difficult.